packages/ui/src/components/atoms/Link.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC, HTMLAttributes } from 'react'
+import styled from 'styled-components'
+import { colors, dimensions, font } from '../../styles'
+
+const StyledLink = styled.a.withConfig<TLink>({
+  shouldForwardProp: (prop) => !['weight'].includes(prop),
+})`
+  /* margin: ${dimensions.spacing.xxs}; */
+  gap: ${dimensions.spacing.xxs};
```

Comment:
Para que tiene Link esto?

Reply:
En consola salía el error de "Preventing Props From Being Passed to the DOM" y decía de usar shouldForwardProps. Además tengo que borrar la línea 8 comentada.